### PR TITLE
Cades build

### PR DESCRIPTION
**Pull request by mcbennet**
## Proposed changes

 Updating ORNL CADES build script

1. An import of python is needed given recent transition to python3
2. Builds with and without AVX are included for use with skylake and
batch nodes, respectively.

## What type(s) of changes does this code introduce?

- Build related changes

### Does this introduce a breaking change?

- No

## What systems has this change been tested on?

ORNL CADES

## Checklist

_Update the following with a yes where the items apply. If you're unsure about any of them, don't hesitate to ask.  This is
simply a reminder of what we are going to look for before merging your code._

- Yes. This PR is up to date with current the current state of 'develop'
- Yes. Code added or changed in the PR has been clang-formatted
- No. This PR adds tests to cover any new code, or to catch a bug that is being fixed
- No. Documentation has been added (if appropriate)

### CODE REVIEW

File: config/build_ornl_cades.sh [hunk 1]
```diff
@@ -37,47 +37,50 @@ CMAKE_FLAGS="-DCMAKE_C_COMPILER=mpicc \
...
 
-# Configure and build cpu real AoS
+# Configure and build cpu real SoA
 echo ""
 echo ""
 echo "building QMCPACK for cpu AoS real for CADES SHPC Condo"
-mkdir -p build_cades_cpu_real
-cd build_cades_cpu_real
-cmake -DENABLE_SOA=0 $CMAKE_FLAGS ..
+mkdir -p build_cades_cpu_real_SoA
```
Comment (jtkrogel) [inline config/build_ornl_cades.sh]: Probably better to call these "_skylake" rather than "_SoA".

File: config/build_ornl_cades.sh [hunk 2]
```diff
@@ -37,47 +37,50 @@ CMAKE_FLAGS="-DCMAKE_C_COMPILER=mpicc \
...
+CMAKE_FLAGS="-DCMAKE_C_COMPILER=mpicc \
+             -DCMAKE_CXX_COMPILER=mpicxx"
+
+# Configure and build cpu real SoA without AVX
 echo ""
 echo ""
 echo "building QMCPACK for cpu SoA real for CADES SHPC Condo"
-mkdir -p build_cades_cpu_real_SoA
-cd build_cades_cpu_real_SoA
+mkdir -p build_cades_cpu_real_SoA_noavx
```
Comment (jtkrogel) [inline config/build_ornl_cades.sh]: Better to name as "_batch" rather than "_SoA_noavx".

Comment (mcbennet) [reply config/build_ornl_cades.sh]: Just to confirm, remove both "SoA" and "noavx" qualifiers and add "batch" ?

Comment (ye-luo) [reply config/build_ornl_cades.sh]: In this case, you still get avx2. noavx512 is more accurate

**Comment by prckent** [comment]
Since there are many queues and various generations of hardware on CADES, it would be beneficial to very clearly state which queues and CADES units this targets in comments at the top of the file. e.g. CNMS nodes are haswells and broadwells.

**Comment by jtkrogel** [comment]
There is only one queue that benefits from the AVX instructions (skylake).  All the others have AVX removed.  Correct?

Perhaps best then to have "qmcpack_cpu_real" and "qmcpack_cpu_real_skylake"?

**Comment by prckent** [comment]
As the main users of the script, pick what you find logical. Should other people express an interest we can update.

File: config/build_ornl_cades.sh (hunk 1 above)
Comment (jtkrogel) [reply config/build_ornl_cades.sh]: Keep skylake.

File: config/build_ornl_cades.sh (hunk 2 above)
Comment (jtkrogel) [reply config/build_ornl_cades.sh]: Remove all qualifiers here.

**Comment by jtkrogel** [comment]
Sounds good.

**Comment by markdewing** [comment]
okay to test

**Pull request merged**